lineman/app/components/group_page/group_page_controller.coffee

```diff
@@ -38,7 +38,7 @@ angular.module('loomioApp').controller 'GroupPageController', ($rootScope, $loca
     ModalService.open LogoPhotoForm, group: => @group
 
   @handleSubscriptionSuccess = ->
-    if AppConfig.chargify and $location.search().chargify_success?
+    if (AppConfig.chargify or AppConfig.environment != 'production') and $location.search().chargify_success?
```

Comment:
Would you be ok with removing the chargify check then? It's impossible to
test this otherwise.

On Wed, 11 Nov 2015 at 18:55 James Kiesel <notifications@github.com> wrote:

> In lineman/app/components/group_page/group_page_controller.coffee
> <https://github.com/loomio/loomio/pull/2702#discussion_r44503821>:
>
> > @@ -38,7 +38,7 @@ angular.module('loomioApp').controller 'GroupPageController', ($rootScope, $loca
> >      ModalService.open LogoPhotoForm, group: => @group
> >
> >    @handleSubscriptionSuccess = ->
> > -    if AppConfig.chargify and $location.search().chargify_success?
> > +    if (AppConfig.chargify or AppConfig.environment != 'production') and $location.search().chargify_success?
>
> I'd rather not change behaviour based on environment here.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/loomio/loomio/pull/2702/files#r44503821>.
>